### ISSUE TITLE
Query log: low contrast in previous button

### ISSUE THREAD

**Issue by reynir** [robur-coop/dnsvizor#84]
> The contrast between the small text and background is not sufficient. The contrast ratio is 4.39:1 when it should be 4.5:1 for text smaller than 18px.

<img width="107" height="57" alt="Image" src="https://github.com/user-attachments/assets/98fe3483-6b4d-49e3-9bd0-839ec8d1d7ce" />

[1.4.3 Contrast (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum)
